lib/model/frames/form.js

```diff
@@ -140,11 +140,16 @@ Form.Partial = class extends Form {};
 
 Form.Extended = class extends Frame.define(
   'submissions',        readable,       'lastSubmission', readable,
+  'received',           readable,       'has_issues',       readable,
```

Comment:
Just a random comment, I think it'd be great to name the field `hasIssues` rather than `has_issues`, since other fields are camelCase.